kvs/src/lib.rs

```diff
@@ -0,0 +1,687 @@
+//! # Key-Value-Storage API and Implementation
+//!
+//! ## License
+//!
+//! Copyright (c) 2025 Qorix GmbH
+//!
+//! This program and the accompanying materials are made available under the terms of the Apache
+//! License, Version 2.0 which is available at https://www.apache.org/licenses/LICENSE-2.0.
+//!
+//! SPDX-License-Identifier: Apache-2.0
+//!
+//! ## Introduction
```

Comment:
The License should always be the very first element on every file. Ideally we stick to the Eclipse copyright information if we contribute to S-CORE directly instead of our own repo.

```suggestion
//! Copyright (c) 2024 Contributors to the Eclipse Foundation
//!
//! See the NOTICE file(s) distributed with this work for additional
//! information regarding copyright ownership.
//! 
//! This program and the accompanying materials are made available under the
//! terms of the Apache License Version 2.0 which is available at
//! https://www.apache.org/licenses/LICENSE-2.0
//! 
//! SPDX-License-Identifier: Apache-2.0

//! # Key-Value-Storage API and Implementation
//! ## Introduction
```

Reply:
Replaced the license in every code file.